packages/remix-adapter/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export type { GetLoadContextFunction, RequestHandler } from './server'
```

Comment:
I think we can't forget to install the globals

```suggestion
import './globals'

export type { GetLoadContextFunction, RequestHandler } from './server'
```

`globals.ts`:
```ts
import { installGlobals } from '@remix-run/node'

installGlobals()
```